classroom.rb

```diff
@@ -0,0 +1,8 @@
+class Classroom
+  has_many :students
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+  end
```

Comment:
- [ ] Kindly add a method that serves as the 'has-many' side (a classroom has many students).